tools/clang/test/SemaHLSL/attributes/wavesize_range_lib_6_6.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: %dxc -E main -T lib_6_6 %s -verify
+
+[wavesize(4)]
+[numthreads(1,1,8)]
+[shader("compute")]
+void one() {
+}
+
+[wavesize(4, 8)] // expected-error{{wavesize attribute takes no more than 1 argument}}
```

Comment:
This doesn't seem right:
1. It's not true: WaveSize can take more than 1 argument.
2. Is no preferred size really an invalid form?

Reply:
Ah, this is because the shader model is 6.6.  In that case, we should report that it requires SM 6.8, not that the attribute doesn't take more than 1 argument.

Reply:
Under SM 6.6?

Reply:
Shouldn't wavesize take exactly one argument given the shader model here is 6.6?